### ISSUE TITLE
Domoticz Api error

### ISSUE THREAD

**Issue by odelma** [stas-demydiuk/domoticz-plugins-manager#8]
Hi,

the custom page does not load a list of plugins and the console shows an error referring to domoticz Api: No Plugin devices found (row 52):

> function init() {

            return domoticzApi.sendRequest({
                type: 'devices',
                displayhidden: 1,
                filter: 'all',
                used: 'all'
            })
                .then(domoticzApi.errorHandler)
                .then(function(response) {
                    if (response.result === undefined) {
                        throw new Error('No Plugin devices found')
                    }

I cannot see anything wrong with the api as such, it returns a list of devices just fine. What might be the issue? Also, is it possible to manage plugins installed with pp-manager with this plugin? Any other way than uninstalling and reinstalling with this?

Thanks for the great work! 👍

**Comment by frustreermeneer** [comment]
I probably had the same. 
Make sure you accept new hardware devices in the settings. 
Then reinstall the plugin manager.

**Comment by odelma** [comment]
Thanks for the suggestion, but I think this is something else. I have "accept new hardware" enabled continuously and if I e.g. disable and enable the plugin I get:

`2020-10-15 00:33:12.622 (Python Plugins Manager) Uninstalling plugin custom page...
2020-10-15 00:33:12.623 (Python Plugins Manager) Uninstalling plugin custom page completed.
2020-10-15 00:33:12.581 Status: (Python Plugins Manager) Stop directive received.
2020-10-15 00:33:12.581 Status: (Python Plugins Manager) Exiting work loop.
2020-10-15 00:33:12.881 Status: (Python Plugins Manager) Stopping threads.
2020-10-15 00:33:12.881 Status: (Python Plugins Manager) Stopped.
2020-10-15 00:33:18.300 (Python Plugins Manager) Debug logging mask set to: NONE
2020-10-15 00:33:18.327 (Python Plugins Manager) Installing plugin custom page...
2020-10-15 00:33:18.329 (Python Plugins Manager) Installing plugin custom page completed.
2020-10-15 00:33:18.100 Status: (Python Plugins Manager) Started.
2020-10-15 00:33:18.298 Status: (Python Plugins Manager) Entering work loop.
2020-10-15 00:33:18.298 Status: (Python Plugins Manager) Initialized version 1.0.0`

But still the same issue. What is strange, it used to work just or at the beginning (a few weeks back). Then there was a long period I did not need it and now this. Meanwhile only changes are updated Domoticz and a different theme in use. But I also tried it with the default theme, just in case...
